### PR TITLE
Improve docs, fixes #1

### LINKED ISSUE #1: Better documentation.
This repo could really use some better documentation please!!!!!!

### PR DESCRIPTION
**Pull request by dlmather**
Sorry about the slow turnaround guys, was busy working on my private fork, reposito.ry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
 # repository
+Repository is the repository to end all repositories. Builds in seconds. No dependancies. Every line of code extensively covered. This is the only repository you will ever need.
+
+## Installation
+It couldn't be easier.
+
+`git clone https://github.com/JDvorak/repository.git`
```
Comment (kgryte) [inline README.md]: Prefer as code block rather than `inline`.

```
$ git clone https://github.com/JDvorak/repository.git
```

**Comment by kgryte** [comment]
Can we get some code coverage statistics? Maybe include a couple of badges; e.g., `travis-ci`, `codecov`, and maybe provide a badge to indicate whether the repository is `stable` or `experimental`?

**Comment by kgryte** [comment]
@dlmather Thanks for your hard work!

File: README.md (hunk 1 above)
Comment (dlmather) [inline README.md]: Thanks for bringing this to my attention.
https://github.com/dlmather/repository/commit/8e6048c5f97f3c9971f6d9f6392ee5f2859e95d5

**Comment by JDvorak** [comment]
GREAT JOB TEAM

**Comment by kgryte** [comment]
Alas, @dlmather's Markdown does not adhere to the Commonmark spec. Headings are block elements; thus, an empty line should separate a heading from paragraphs.

**Comment by kgryte** [comment]
Also, the code fence is missing a language annotation.

**Comment by JDvorak** [comment]
Good catch.

@dlmather I'll be rolling your changes back until you fix. 

Sorry to the community for this disruption, thankfully it was caught before I published to npm.

**Comment by JDvorak** [comment]
@dlmather Let me know when you get that fixed. I'll merge it in, speed is my number one priority when it comes to merging in code from the community.

**Pull request merged**